contributor-book/plugins.md

```diff
@@ -44,8 +44,8 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-nu-plugin = "0.80.0" # These version numbers may differ
-nu-protocol = "0.80.0"
+nu-plugin = "0.84.0" # These version numbers may differ
+nu-protocol = "0.84.0"
```

Comment:
maybe we can just wait for tomorrow to bump this to `0.85.0` directly?

Reply:
Bumped nu-plugin and nu-protocol dependant crates in sample Cargo.toml to version 0.85.0 each.